app/models/user_team.rb

```diff
@@ -5,4 +5,7 @@ class UserTeam < ApplicationRecord
 
   belongs_to :user
   belongs_to :team, foreign_key: 'group_id', inverse_of: :user_teams
+
+  validates_uniqueness_of :user_id, scope: :group_id, on: :create,
```

Comment:
**comment (non-blocking):** one thought I had is that if someone tries to add a user that's already on the team we don't return an error, just proceed with a warning (since it's not a risky request, just redundant). That said, I don't think we should tackle that in this ticket but consider it for consideration as a future improvement. Let me know what you think!